spec/spec_helper.rb

```diff
@@ -84,5 +84,11 @@
 #   # as the one that triggered the failure.
 #   Kernel.srand config.seed
 
-  config.filter_run_excluding :check_links unless !!ENV['RSPEC_CHECK_LINKS']
+  if !!ENV['RSPEC_SKIP_LINK_CHECK']
```

Comment:
ah.. it's not i guess.

However, it would be nice to avoid confusion by allowing some "No" values to be recognized. For instance, I would want to be able to ```RSPEC_SKIP_LINK_CHECK=false|no|0``` and have it **not skip** the link checks.

Do you know of a good idiom for that? Anyway, it can wait for a new PR. Will remove the unnecessary `!!` and resubmit. thanks